docs/docsite/rst/playbook_guide/playbooks_reuse_roles.rst

```diff
@@ -108,15 +124,21 @@ The classic (original) way to use roles is with the ``roles`` option for a given
         - common
         - webservers
 
-When you use the ``roles`` option at the play level, for each role 'x':
+When you use the ``roles`` option at the play level, each role 'x' looks for a main.yml/main.yaml/main by default (you can specify otherwise with include/import_role) in the following directories:
```

Comment:
```suggestion
When you use the ``roles`` option at the play level, each role 'x' looks for a ``main.yml/main.yaml/main`` by default (you can specify otherwise with ``include/import_role``) in the following directories:
```